### PR TITLE
Add igroup attribute to container interface

### PR DESCRIPTION
**Pull request by kotaaaa**
## Overview

- Added IGroup attribute to IContainer interface instead of using groupId, groupName directly.

## Changes

- Added IGroup attribute to IContainer interface
- Corrected to reference IContainer's IGroup.
- Name convention of from groupId to id, groupName to name

## Assignee Checklist:

- [x] The base branch is correct (See: [Types of Branches](https://github.com/genesis-tech-tribe/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#types-of-branches))
- [x] The branch name follows the [Branch Naming Conventions](https://github.com/genesis-tech-tribe/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#branch-naming-conventions)
- [x] The correct assignees and reviewers have been designated for this PR
- [x] The coding style follows the [Coding Style Guide](https://github.com/genesis-tech-tribe/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [x] All the related issues are associated with this PR
- [x] All criteria in the associated issues are met (please tick the checkboxes)
- [x] My changes do not generate new warnings or errors (especially in the console)

## Reviewer Checklist:

- [x] The code follows the generic best practices and our [Coding Style Guide](https://github.com/genesis-tech-tribe/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [x] Code is well-commented and easy to understand
- [x] UI changes accurately reflect the provided design

### CODE REVIEW

**Comment by uskayyyyy** [comment]
@kotaaaa There is an error in build time. I am not sure why the status check is not triggered... Could you try a build locally and fix the error please?

File: src/features/foods/components/FoodSort.tsx
Comment (uskayyyyy) [inline src/features/foods/components/FoodSort.tsx:1]: Wrong icon

**Review by uskayyyyy — CHANGES_REQUESTED** [review]
@kotaaaa 
Thank you for updating! Please take a look at the comment.

File: src/features/foods/components/FoodSort.tsx
Comment (kotaaaa) [reply src/features/foods/components/FoodSort.tsx:1]: Thank you, fixed.

**Review by uskayyyyy — APPROVED** [review]
Thank you!

**Pull request merged**